tidalapi/bin/generate-api-config.json

```diff
@@ -0,0 +1,21 @@
+{
+  "inputs": [
+    {
+      "inputURL": "https://developer.tidal.com/apiref/api-specifications/api-public-user-jsonapi/tidal-user-v2-openapi-3.0.json"
+    },
+    {
+      "inputURL": "https://developer.tidal.com/apiref/api-specifications/api-public-catalogue-jsonapi/tidal-catalog-v2-openapi-3.0.json"
+    },
+    {
+      "inputURL": "https://developer.tidal.com/apiref/api-specifications/api-public-search-jsonapi/tidal-search-v2-openapi-3.0.json"
+    },
+    {
+      "inputURL": "https://developer.tidal.com/apiref/api-specifications/api-public-user-content/tidal-user-content-openapi-3.0.json"
+    }
+
+  ],
+  "output": "./tidal-api-joined.json",
+  "openapi_generator_path": "tbd"
```

Comment:
This would point to where the executable would be found. As I still work on providing it, it is tbd for now.